test/mode/modeNormal.test.ts

```diff
@@ -1805,6 +1805,20 @@ suite('Mode Normal', () => {
     end: ['blah blah', '|blah blah'],
   });
 
+  newTest({
+    title: 'can do [count]Y',
+    start: ['|one', 'two', 'three'],
+    keysPressed: '2Yp',
+    end: ['one', '|one', 'two', 'two', 'three'],
+  });
+
+  newTest({
+    title: 'can do [count]Y if count is larger than EOF',
+    start: ['|one', 'two', 'three'],
+    keysPressed: '100Yp',
+    end: ['one', '|one', 'two', 'three', 'two', 'three'],
+  });
+
```

Comment:
Better to add a test case for >100 lines by:
```javascript
lines = new Array(100).fill().map((_, i) => `line-${i}`)
```
Because the issue is `error in "[count]Y/J" with large count`, now we should prove for large count it works for either <=100 or > 100.

Reply:
Thank you for the review!
I think it's right in a way, but I think that the problem is that the pull request title and commit comments are not right. The root issue is that error occurred when exceeding the max number of rows.

Reply:
Ah, make sense.

Reply:
Thank you for confirmation, I changed PR title.